pkg/api/roles.go

```diff
@@ -137,29 +137,29 @@ func (hs *HTTPServer) declareFixedRoles() error {
 		Grants: []string{string(models.ROLE_VIEWER)},
 	}
 
-	currentOrgReaderRole := accesscontrol.RoleRegistration{
+	orgReaderRole := accesscontrol.RoleRegistration{
 		Role: accesscontrol.RoleDTO{
-			Version:     4,
-			Name:        "fixed:current.org:reader",
-			DisplayName: "Current Organization reader",
-			Description: "Read the current organization, such as its ID, name, address, or quotas.",
+			Version:     5,
+			Name:        "fixed:organization:reader",
+			DisplayName: "Organization reader",
+			Description: "Read an organization, such as its ID, name, address, or quotas.",
 			Group:       "Organizations",
 			Permissions: []accesscontrol.Permission{
 				{Action: ActionOrgsRead},
 				{Action: ActionOrgsQuotasRead},
 			},
 		},
-		Grants: []string{string(models.ROLE_VIEWER)},
+		Grants: []string{string(models.ROLE_VIEWER), accesscontrol.RoleGrafanaAdmin},
 	}
 
-	currentOrgWriterRole := accesscontrol.RoleRegistration{
+	orgWriterRole := accesscontrol.RoleRegistration{
 		Role: accesscontrol.RoleDTO{
-			Version:     4,
-			Name:        "fixed:current.org:writer",
-			DisplayName: "Current Organization writer",
-			Description: "Read the current organization, its quotas, or its preferences. Update the current organization properties, or its preferences.",
+			Version:     5,
+			Name:        "fixed:organization:writer",
+			DisplayName: "Organization writer",
+			Description: "Read an organization, its quotas, or its preferences. Update an organization properties, or its preferences.",
```

Comment:
This sounds a bit awkward to me. Maybe without the "an" in the second sentence?
```suggestion
			Description: "Read an organization, its quotas, or its preferences. Update organization properties, or its preferences.",
```